docs/tutorials/civo.md

```diff
@@ -0,0 +1,187 @@
+# Setting up ExternalDNS for Services on Civo
+
+This tutorial describes how to setup ExternalDNS for usage within a Kubernetes cluster using Civo DNS Manager.
+
+Make sure to use **>=1.7.1** version of ExternalDNS for this tutorial.
+
+## Managing DNS with Civo
+
+If you want to learn about how to use Civo DNS Manager read the following tutorials:
+
+[An Introduction to Managing DNS](https://www.civo.com/learn/configure-dns)
+
+## Get Civo Token
+
+Copy the token in the settings fo your account
+
+The environment variable `CIVO_TOKEN` will be needed to run ExternalDNS with Civo.
+
+## Deploy ExternalDNS
+
+Connect your `kubectl` client to the cluster you want to test ExternalDNS with.
+Then apply one of the following manifests file to deploy ExternalDNS.
+
+### Manifest (for clusters without RBAC enabled)
+
+```yaml
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: external-dns
+spec:
+  strategy:
+    type: Recreate
+  selector:
+    matchLabels:
+      app: external-dns
+  template:
+    metadata:
+      labels:
+        app: external-dns
+    spec:
+      containers:
+      - name: external-dns
+        image: k8s.gcr.io/external-dns/external-dns:v0.7.3
```

Comment:
This is also wrong, it's fine, you don't need to get the image right, but please indicate this clearly in the tutorial. Also, `k8s.gcr.io` is deprecated, please look at the latest releases for the right URL for the docker registry.